openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -65,3 +65,31 @@ def edx_course_enrollment_mode_changed(current_event, caliper_event):
 
     caliper_event['actor']['type'] = 'Person'
     return caliper_event
+
+
+def edx_course_enrollment_upgrade_clicked(current_event, caliper_event):
+    """
+    The server emits an enrollment.upgrade_clicked event when the student
+    clicks on Upgrade Enrollment Button.
+
+    :param current_event: default log
+    :param caliper_event: log containing both basic and default attribute
+    :return: final created log
+    """
+
+    caliper_object = {
+        'id': current_event['page'],
+        'type': current_event['event_source'],
```

Comment:
kindly remove 'type' for now, for object fields we will be using other method afterwards.